src/components/PrizePage/Prizes.js

```diff
@@ -81,35 +121,66 @@ export default function Prizes() {
 	const hideImg = useMediaQuery(theme.breakpoints.down('xs'));
 
 	return (
-		<Container maxWidth='md'>
-			<TableContainer>
-				<Table aria-label="prize table">
-					<TableHead>
-						<TableRow>
-							{ hideImg ? null : <TableCell></TableCell> }
-							<TableCell align='center' className={classes.header}>Track</TableCell>
-							<TableCell align='center' className={classes.header}>Prize</TableCell>
-						</TableRow>
-					</TableHead>
-					<TableBody>
-						{allPrizes.map(row =>
-							<TableRow key={row.track}>
-								{ hideImg ?
-									null :
-									<TableCell align='center' width='20%'>
-										<img src={row.img} height='60px'/>
+		<>
+			<Container maxWidth='md'>
```

Comment:
question: do we need the Container if we have a `Grid container` already with the same maxWidth?

Reply:
I couldn't get it to work (have any effect) applying it to `Grid container`.